sal_interfaces/ATPneumatics/ATPneumatics_Commands.xml

```diff
@@ -89,13 +89,13 @@
         <Property/>
         <Action/>
         <Value/>
-        <Description>Open the M1 mirror covers. The command is reported done right away, before the covers finish moving.</Description>
+        <Description>Open the M1 mirror covers. Note the command is reported done right away, before the covers finish moving.</Description>
```

Comment:
Are you sure it is actually the case that the command is reported as done right away? If so, we probably want this changed in the CSC. If I remember correctly, all commands where being handled this way and I asked Rolando to fix them. Maybe this one was left behind and we need to ask him to fix it.

Reply:
Unsure of current status. Changed to what the performance should be. WIll have the ATMCS fixed if non-compliant.

Reply:
Not sure I understood... the expected behavior should be that the command is ack once it finishes opening. I think the Description should only read "Open the M1 mirror covers."

Reply:
I had just missed this. In theory it should be fixed...